streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -197,22 +199,24 @@
     private static final Logger log = LoggerFactory.getLogger(TopologyTestDriver.class);
 
     private final Time mockWallClockTime;
-    private final InternalTopologyBuilder internalTopologyBuilder;
+    private InternalTopologyBuilder internalTopologyBuilder;
```

Comment:
Needed to change the constructor to make it shorter (checkstyle failed). Hence, some variables cannot be `final` any longer.